alf/metrics/metrics_test.py

```diff
@@ -68,12 +69,17 @@ def _create_trajectories(self):
         [('testEnvironmentStepsGraph', EnvironmentSteps, 5, 6),
          ('testNumberOfEpisodesGraph', NumberOfEpisodes, 4, 2),
          ('testAverageReturnGraph', AverageReturnMetric, 6, 9.0),
+         ('testAverageDiscountedEpisodicReturnMetric',
+          AverageDiscountedEpisodicReturnMetric, 6, 8.945),
          ('testAverageEpisodeLengthGraph', AverageEpisodeLengthMetric, 6, 2.0),
          ('testAverageEnvInfoMetric', AverageEnvInfoMetric, 6,
           dict(x=torch.as_tensor(5.), y=torch.as_tensor(1.5)))])
     def testMetric(self, metric_class, num_trajectories, expected_result):
         trajectories = self._create_trajectories()
-        if metric_class in [AverageReturnMetric, AverageEpisodeLengthMetric]:
+        if metric_class in [
+                AverageReturnMetric, AverageDiscountedEpisodicReturnMetric,
+                AverageEpisodeLengthMetric
+        ]:
```

Comment:
I'd be happier if a test for multi dim reward could be added, since this change handles multi dim reward as well.

Reply:
Added